### PR TITLE
docs: add a section on getting the client working with celery

### PR DESCRIPTION
**Pull request by sighphyre**
Adds documentation on how to get this project working with Celery

### CODE REVIEW

File: docs/celery.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+****************************************
+Running with Celery
+****************************************
+
+The client is also compatible with Celery, however, the initialize_client method needs to be called in the worker_process_init event, otherwise toggles will not update correctly when changes on the server.
```
Comment (thomasheartman) [inline docs/celery.rst:5]: Nice, but maybe we could rephrase it to something like this?

```suggestion
To use this client with Celery, call the `initialize_client` method in the `worker_process_init` event. If you don't initialize the client inside the `worker_process_init` event, toggles will not update correctly when changes occur on the server.
```

What does "update correctly" mean here? Will they not update at all or just at weird times?

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Good one! 🙌🏼  Should be great to get this in 😄  I've added a small rewording suggestion and have a little discussion point before we do, though.

File: docs/celery.rst (hunk 1 above)
Comment (sighphyre) [reply docs/celery.rst:5]: Oh that's much better, thanks for the suggestion. Not "update correctly" should probably be "not update at all" 

What about this (just removing the "correctly"):

To use this client with Celery, call the `initialize_client` method in the `worker_process_init` event. If you don't initialize the client inside the `worker_process_init` event, toggles will not update when changes occur on the server.

Comment (thomasheartman) [reply docs/celery.rst:5]: Yeah, that's great! Might even wanna **bold** or _italicize_ the "not" in "will not update". But maybe it can be reworded a bit. Something like:

> If you don't initialize the client inside the `worker_process_init` event, the client will not poll for updates. As such, the client will not update its feature toggles even when changes are made server side.
